app/controllers/EmailController.java

```diff
@@ -19,13 +20,17 @@
 @Controller("emailController")
 public class EmailController extends BaseController {
 
+    private BridgeConfig bridgeConfig;
```

Comment:
I can make that quick adjustment as you are suggesting.  Still the fundamental problem is dependency injection in an abstract class, which is a big no-no.  An abstract bean (<bean abstract="true" />) is probably the original intention which is to share common properties.  But a bean marked as "abstract" is totally different from an abstract class.